charts/network-agent/templates/daemonset.yaml

```diff
@@ -61,6 +58,11 @@ spec:
               valueFrom:
                 fieldRef:
                   fieldPath: metadata.name
+            {{- with .Values.extraEnvVars }}
+            {{- toYaml . | nindent 12 }}
+            {{- end }}
+          resources:
```

Comment:
We should only render `resources:` if the user has set `.Values.resources`.  You can do something like `{{- if .Values.resources }}`

Reply:
Oh good catch! Updated in https://github.com/honeycombio/helm-charts/pull/297/commits/9dacca5c255ab74f97f4039c5861846ecab7c52c and after this update I can see in a `--dry-run` that it no longer shows in the final daemonset.